bot.py

```diff
@@ -94,9 +94,10 @@ def log_exception(exception=None):
         title = match.group(1).strip(" *.\n")
         report_num = get_next_report_num(identifier)
         report_id = f"{identifier}-{report_num}"
+        attach = "\n" + '\n'.join(f"\n![{attachment.filename}]({attachment.url})" for attachment in message.attachments)
```

Comment:
What about attachments that aren't images?

Reply:
Displays a link with a broken image icon. (https://github.com/croebh-test/repo-1/issues/7) 

I just added a check for file extensions of gif, png, and jpg, and only add the `!` before the link markdown.